pkg/plugins/backendplugin/grpcplugin/client.go

```diff
@@ -40,7 +43,20 @@ var pluginSet = map[int]goplugin.PluginSet{
 	},
 }
 
-func newClientConfig(executablePath string, args []string, env []string, skipHostEnvVars bool, logger log.Logger,
+type clientTracerProvider struct {
```

Comment:
So in order to pass the tracer with `WithTracerProvider` we need to satisfy the `TracerProvider` interface of opentelemetry. Thus struct and methods are added